db.js

```diff
@@ -220,33 +227,54 @@ export class ActivityPubDB extends EventTarget {
     await tx.done()
   }
 
-  async * searchNotes ({ attributedTo } = {}, { skip = 0, limit = DEFAULT_LIMIT, sort = -1 } = {}) {
+  async * searchNotesRandom ({ limit = DEFAULT_LIMIT } = {}) {
```

Comment:
I think you accidentally deleted the searchNotes function. Mind re-adding it and just updating searchNotesRandom.

Alternately maybe we can replace the `sort` argument with `0` or something to indicate `random` and merge the functions?